cumulus/parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs

```diff
@@ -238,6 +262,7 @@ pub type AssetTransactors = (
 	PoolFungiblesTransactor,
 	UniquesTransactor,
 	ERC20Transactor,
+	ERC721Transactor,
```

Comment:
Originally I thought putting the two transactors one after the other wouldn't work because they both match, so the first one would be used. However, when the first one fails because the underlying contract doesn't implement the ERC721 interface, then it will try with the next one and succeed. The tests show this does indeed work 👍